containers/base/Dockerfile

```diff
@@ -141,7 +145,7 @@ RUN ipython profile create default && \
     if [ -d /datalab/lib/pydatalab/.git ]; then \
         echo "use local lib"; \
       else \
-        git clone https://github.com/googledatalab/pydatalab.git /datalab/lib/pydatalab; \
+        git clone https://github.com/googledatalab/pydatalab.git  --single-branch --branch feature/ml /datalab/lib/pydatalab; \
```

Comment:
We should get rid of this line of change too; we want to keep the master branch.